VisualPinball.Engine/VPT/Mappings/Mappings.cs

```diff
@@ -15,9 +15,12 @@
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
 using System.Linq;
 using System.Text.RegularExpressions;
+using NetVips;

```

Comment:
Are `NetVip` and `System.Diagnostics` really needed?

Reply:
nope, cleaning.